open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -61,47 +61,45 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        # return column_value_mapping
</pre><p>remove this if it is not used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
ok，Thanks for reminding